src/pages/en/guides/deploy/sst.mdx

```diff
@@ -0,0 +1,23 @@
+---
+title: Deploy your Astro Site to AWS with SST
+description: How to deploy your Astro site to AWS with SST
+layout: ~/layouts/DeployGuideLayout.astro
+i18nReady: true
+---
+
+You can deploy an Astro site using [SST](https://sst.dev), an open-source framework for deploying applications to AWS. This is the easiest way to get a fully serverless Astro deployment on AWS with SSR support.
+
+You can also use any additional SST constructs like Cron Jobs, Buckets, Queues, etc while maintaining typesafety.
```

Comment:
We are using "type-safety" across the docs rather than typesafety or "type safety", I'd change this to the former for consistency.
```suggestion
You can also use any additional SST constructs like Cron Jobs, Buckets, Queues, etc while maintaining type-safety.
```